src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Figure circle = new Circle();
+        Figure rectangle = new Rectangle();
+        Figure rightTriangle = new RightTriangle();
+        Figure square = new Square();
+        Figure isoscelesTrapezoid = new Figure();
+        Figure[] figures = new Figure[]{circle, rectangle, rightTriangle, square, isoscelesTrapezoid};
+        int count = 1;
+        FigureSupplier figure = new FigureSupplier();
+        Figure[] allFigures = new Figure[]{figure.getRandomFigure(figures), figure.getRandomFigure(figures), figure.getRandomFigure(figures),
```

Comment:
It's not necessary to create an array of random figures in this way, and it also reduces the readability of the code. Consider using a loop to fill the array with random figures.